packages/fhir-converter/src/templates/cda/Sections/Notes.hbs

```diff
@@ -1,7 +1,8 @@
-{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.65')}}
-    {{#with (getAllCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.65') as |sections|}}
+{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.65' '1.3.6.1.4.1.19376.1.5.3.1.3.5')}}
+    {{#with (getAllCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.65' '1.3.6.1.4.1.19376.1.5.3.1.3.5') as |sections|}}
+            {{!-- sections here is already an array so this is not a bug --}}
             {{#each sections}}
-                {{#each (toArray 2_16_840_1_113883_10_20_22_2_65.entry) as |note|}}
+                {{#each (multipleToArray this.[2_16_840_1_113883_10_20_22_2_65].entry this.[1_3_6_1_4_1_19376_1_5_3_1_3_5].entry) as |note|}}
```

Comment:
DiagnosticReport: 27459 => 30906
Organization: 20506 => 20514
Practitioner: 40036 => 40061

More diagnostic reports, and each diagnostic reports entails the creation of more practitioners and organizations.